### ISSUE TITLE
browserslist location is documented wrong

### ISSUE THREAD

**Issue by josacar** [ai/autoprefixer-rails#93]
In README.md says:

If you need to specify browsers for your Rails project, you can save them to:

- `browserslist` and place it under `app/assets/stylesheets/` or any of its ancestor directories

But in code this is not the place to start looking. Let's take a look to `lib/autoprefixer-rails/processor.rb`: 

```ruby
module AutoprefixerRails
  class Processor
    # ...
    
    private

    def params_with_browsers(from = nil)
      unless from
        if defined? Rails and Rails.respond_to?(:root) and Rails.root
          from = Rails.root.join('app/assets/stylesheets').to_s
        else
          from = '.'
        end
      end

      params = @params
      if not params.has_key?(:browsers) and from
        config = find_config(from)
        if config
          params = params.dup
          params[:browsers] = parse_config(config)
        end
      end

      params
    end
    
    # ...

    def find_config(file)
      path = Pathname(file).expand_path.dirname

      while path.parent != path
        config = path.join('browserslist')
        return config.read if config.exist? and not config.directory?
        path = path.parent
      end

      nil
    end
    
    # ...
  end
end
```

The code starts looking from `app/assets/stylesheets/` but in `find_config` method stylesheets is popped out the path as `dirname` is used:

```
path = Pathname(file).expand_path.dirname
```

So the right location should be `app/assets` instead of `app/assets/stylesheets` according to the implementation.

**Comment by ai** [comment]
Good catch. Do your want to send PR?

**Comment by ai** [comment]
Released in 6.3.7
